### ISSUE TITLE
Kotlinx-html 0.8.0 mpp artifact is missing on maven central

### ISSUE THREAD

**Issue by SPC-code** [Kotlin/kotlinx.html#201]
Platform artifacts are there, but not MPP without suffix.

**Comment by e5l** [comment]
Fixed with 0.8.1

**Comment by digitalheir** [comment]
...But 0.8.1-jvm does not exist?

```
:liblawlyhtml:test: Could not find org.jetbrains.kotlinx:kotlinx-html-jvm:0.8.1
```
